python-sdk/src/astro/databases/base.py

```diff
@@ -455,6 +455,13 @@ def load_file_to_table(
                 chunk_size=chunk_size,
             )
 
+    @staticmethod
+    def get_dataframe_from_file(file: File):
+        """Get pandas dataframe file
+        :param file: File path and conn_id for object stores
+        """
+        return file.export_to_dataframe()
```

Comment:
This definitely feels out of place here.

Can you add a comment on why this is placed in DB for future reference

Reply:
@kaxil Added a comment explaining the need for two different paths.